examples/official-storybook/stories/addon-viewport.stories.js

```diff
@@ -18,10 +20,26 @@ storiesOf('Addons|Viewport.Custom Default (Kindle Fire 2)', module)
       I've inherited <b>Kindle Fire 2</b> viewport from my parent.
     </Panel>
   ))
-  .add('Overridden', () => (
-    <Viewport name="iphone6">
```

Comment:
Does `Viewport` still work? If so, let's keep a story using it

Reply:
Yeah, it makes sense

Reply:
Done